script/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")/.." || exit
+
+source script/helpers.sh
+
+script/bootstrap.sh || exit
+
+task_inform "Setting up your app..."
+
+subtask_exec "Rehashing RBenv" rbenv rehash
+
+task_inform "Cleaning up..."
+
+subtask_exec "Cleaning up tmp log" rm "$LOGFILE"
+
+task_inform "Checking setup..."
+
+echo "Your app is set up. Navigate to http://localhost:3000 in your browser to see a local running copy."
```

Comment:
should we get rid of this since the setup script isn't actually setting up the app? and then we can put in separate instructions about how to clone the app and set it up?

Reply:
@leahgoldberg @agirlnamedsophia what's the intended workflow for folks? how do they get this code on their machines and run it? does macos come with `git`?